setup.cfg

```diff
@@ -15,6 +15,7 @@ classifier =
     License :: OSI Approved :: BSD License
     Operating System :: OS Independent
     Programming Language :: Python
+    Programming Language :: Python :: 3
```

Comment:
Perhaps we should use `python_requires` instead to ensure we do not break the install of someone on Python 2? (https://stackoverflow.com/questions/42238484/prevent-package-from-being-installed-on-old-python-versions)

Reply:
Happy to accept a PR ;)

Reply:
Sure, just double checking that I wasn't missing some argument :)

Reply:
IMHO Python 2 is a dead language and you can pin if you need to.

Reply:
That is of course an acceptable answer as well :)